ambry-api/src/test/java/com.github.ambry/rest/MockRestResponseChannel.java

```diff
@@ -249,7 +249,7 @@ private void setHeader(String headerName, Object headerValue, Event eventToFire)
           onEventComplete(eventToFire);
         } catch (JSONException e) {
           throw new RestServiceException("Unable to set " + headerName + " to " + headerValue,
-              RestServiceErrorCode.ResponseBuildingFailure);
+              RestServiceErrorCode.InternalServerError);
```

Comment:
I'm trying to reduce the number of unique `RestServiceErrorCode` - otherwise we can come up with a code for every new situation.